test/standalone/test-debuglet.js

```diff
@@ -195,11 +205,88 @@ describe(__filename, function(){
     debuglet.start();
   });
 
-  it('should de-activate when the server responds with isDisabled');
+  it('should de-activate when the server responds with isDisabled', function(done) {
```

Comment:
I believe it does because if it didn't deactivate, it would attempt to fetch breakpoints which is an endpoint that nock does not provide causing the test to fail. This is subtle so I've added explicit `fetcherActive_` checks for clarity.